apps/web/src/app/admin/cohorts/[id]/page-client.tsx

```
@@ -1343,84 +1372,128 @@ export function CohortDetailPageClient({
+										<div
+											key={session.id}
+											className={`group relative overflow-hidden rounded-lg border bg-card transition-all duration-200 ${canAddSession ? "cursor-pointer hover:shadow-md" : ""}`}
+											onClick={
+												canAddSession
+													? () => handleEditSession(session)
+													: undefined
+											}
+											role={canAddSession ? "button" : undefined}
+											tabIndex={canAddSession ? 0 : undefined}
+										>
+											{/* Day and Time Header */}
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)